### PR TITLE
WIN smoke test support for nightly package

### PR DESCRIPTION
**Pull request by guyang3532**
Related pr: https://github.com/pytorch/pytorch/pull/39941
test at: https://github.com/pytorch/pytorch/pull/39943

### CODE REVIEW

**Comment by guyang3532** [comment]
@peterjc123

File: windows/internal/install_nightly_package.bat [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+FOR /f %%i in ('python -c "import sys;print(sys.version)"') do set cur_python=%%i
+
+if not %cur_python:~0,3% == %DESIRED_PYTHON% (
+    echo "The Python version has changed to %cur_python%"
+    echo "Probably the package for the version we want does not exist"
+    echo "conda will change the Python version even if it was explicitly declared"
+)
+
+if "%DESIRED_CUDA%" == "cpu" (
+    call conda list torch | grep cuda || exit /b 0
```
Comment (peterjc123) [inline windows/internal/install_nightly_package.bat:42]: nit: Consider replacing `grep` with `findstr`.

Comment (peterjc123) [reply windows/internal/install_nightly_package.bat:42]: Why do you search for cuda here? Could you please explain?

File: windows/internal/install_nightly_package.bat [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	echo "The installed package is built for CUDA, the full package is"
+	call conda list torch
+) else (
+    call conda list torch | grep cuda%CUDA_VERSION% && exit /b 0
```
Comment (peterjc123) [inline windows/internal/install_nightly_package.bat:46]: nit: grep -> findstr

File: windows/internal/install_nightly_package.bat (hunk 1 above)
Comment (guyang3532) [reply windows/internal/install_nightly_package.bat:42]: > nit: Consider replacing `grep` with `findstr`.

ok

Comment (guyang3532) [reply windows/internal/install_nightly_package.bat:42]: > Why do you search for cuda here? Could you please explain?

If 'cuda' is found in torch package name, then it's build for cuda, not for cpu, print the error info and `exit /b 1`, else `exit /b 0`

File: windows/internal/install_nightly_package.bat (hunk 2 above)
Comment (guyang3532) [reply windows/internal/install_nightly_package.bat]: done

**Comment by peterjc123** [comment]
@seemethere

**Comment by peterjc123** [comment]
@seemethere Could you please take a look?

**Pull request merged**